### PR TITLE
Updating csproj to omit generation of .pdb files in release mode

### PR DESCRIPTION
**Pull request by sattpat**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/AWS.Logger.Log4net/AWS.Logger.Log4net.csproj [hunk 1]
```diff
@@ -35,11 +35,13 @@
   <ItemGroup>
     <ProjectReference Include="..\AWS.Logger.Core\AWS.Logger.Core.csproj" />
   </ItemGroup>
-  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.5' ">
-      <PackageReference Include="log4net" Version="2.0.8" />
```
Comment (costleya) [inline src/AWS.Logger.Log4net/AWS.Logger.Log4net.csproj]: Why was this removed?

Comment (sattpat) [reply src/AWS.Logger.Log4net/AWS.Logger.Log4net.csproj]: the version for log4net was common to both netstandard1.5 and net45. So I have removed the framework conditinal

**Comment by snakefoot** [comment]
@normj Why remove the PDB-files? They are very useful when troubleshooting, and the new portable format are very small in size.

https://docs.microsoft.com/en-us/nuget/create-packages/symbol-packages

But if it means you are going to publish a `.snupkg` then fine with me:

https://blog.nuget.org/20181116/Improved-debugging-experience-with-the-NuGet-org-symbol-server-and-snupkg.html
